provider/bitmovin/bitmovin.go

```diff
@@ -742,7 +722,41 @@ func (p *bitmovinProvider) CreatePreset(_ context.Context, preset db.Preset) (st
 		return "", err
 	}
 
-	return svc.Create(preset)
+	presetSummary, err := svc.Create(preset)
+	if err != nil {
+		return "", err
+	}
+
+	if presetSummary.HasVideo() {
+		deInterlace, err := p.api.Encoding.Filters.Deinterlace.Create(model.DeinterlaceFilter{
+			Name:       "deinterlace",
+			AutoEnable: model.DeinterlaceAutoEnable_META_DATA_AND_CONTENT_BASED,
+		})
+		if err != nil {
+			return "", errors.Wrap(err, "creating deinterlace filter")
```

Comment:
We're trying to update our usage of `errors.Wrap`

```suggestion
			return "", fmt.Errorf("creating deinterlace filter: %w", err)